### DISCUSSION TITLE
Language used in Activate User page

### DISCUSSION THREAD

**Discussion by Tomomi-K1** [zitadel/zitadel#8080] [Q&A]
When I create a new user, we set that user's preferredLanguage as 'ja' (Japanese), but my browser language is set as `English`.

I will receive `initialize user` email in Japanese, but `activate user` page that shows up after clicking `initialize user` button is in English.

1. Is language used for `activate user` page is determined by browser setting or user's preferredLanguage?
2. Is there a way to manually set the language used for `activate user` page? 

we are self-hosting Zitadel and using docker container.

activate user page: 
![image](https://github.com/zitadel/zitadel/assets/31785367/946e66e0-9d60-4eed-bce0-49f98dfb1c2e)

**Comment by Tomomi-K1** [comment]
hello. Does anyone know the answer to this?

**Reply by hifabienne** [reply]
Hei. As far as I know, if we do have a preferred language we choose that, if that is not set, we take what is set on the browser

**Reply by Tomomi-K1** [reply]
@hifabienne I tested this but looks like activate user seems to use browser language even though user has preferred language.

**Reply by hifabienne** [reply]
I created an issue out of this discussion
